### ISSUE TITLE
When will the code be open source?

### ISSUE THREAD

**Issue by Ruizhuo-Xu** [maoyunyao/MAMP#1]

**Comment by maoyunyao** [comment]
As soon as the paper appears in the conference proceedings.

**Comment by Ruizhuo-Xu** [comment]
Thank you very much for your answer.

**Comment by Ruizhuo-Xu** [comment]
I am trying to reproduce the results of the paper, could you please provide me with the experimental settings for training the model from scratch in the paper?
![image](https://github.com/maoyunyao/MAMP/assets/110621793/c57db3c6-5a62-472e-8424-714a00103899)
Currently, I am conducting experiments on the NTU RGBD60 dataset, and the model settings are similar to those described in your paper. Adam optimizer was used with a linear increase in learning rate to 1e-4 for the first 5 epochs, followed by cosine decay to 1e-6. A total of 100 epochs were trained without using weight decay or dropout as regularization strategies. However, the recognition accuracy on the X-sub split is only around 78%, which has a significant gap compared to the results presented in your paper. If convenient, I would greatly appreciate it if you could share your experimental settings. Thank you.

**Comment by maoyunyao** [comment]
We are organizing the code to be open-sourced in the next few days, so stay tuned. You will find the answer to your question in the code then.

**Comment by maoyunyao** [comment]
The code is now released. We are verifying the correctness of the training/inference scripts. If you find any problems with the code, please feel free to open an issue or contact us by sending an email to myy2016[AT]mail.ustc.edu.cn.

**Comment by Ruizhuo-Xu** [comment]
> The code is now released. We are verifying the correctness of the training/inference scripts. If you find any problems with the code, please feel free to open an issue or contact us by sending an email to myy2016[AT]mail.ustc.edu.cn.

Thank you for promptly open sourcing the code. If I encounter any issues, I will report them promptly. However, I didn't seem to find the corresponding code and configuration file for training Transformer from scratch in the code repository. I would greatly appreciate it if you could continue to open source this part. Alternatively, providing the relevant configuration files alone would be very helpful for me. Thank you again!

**Comment by maoyunyao** [comment]
We reuse the main_finetune.py for training the Transformer from scratch. You can find the detailed configurations in script_finetune_scratch.sh.

**Comment by Ruizhuo-Xu** [comment]
> We reuse the main_finetune.py for training the Transformer from scratch. You can find the detailed configurations in script_finetune_scratch.sh.

Thank you for reminding me. I overlooked this script file.

**Comment by Ruizhuo-Xu** [comment]
Can you provide pre-trained model checkpoints?

**Comment by maoyunyao** [comment]
Are there any problems with training the model on your side?

**Comment by Ruizhuo-Xu** [comment]
> Are there any problems with training the model on your side?

No problem, but my computing resources are limited, so I would like to use pre-trained checkpoints to perform some experiments quickly.

**Comment by maoyunyao** [comment]
Okay, but we are currently preparing for a deadline on Friday. We will release the model before the weekend at the latest.

**Comment by Ruizhuo-Xu** [comment]
> Okay, but we are currently preparing for a deadline on Friday. We will release the model before the weekend at the latest.
Thank you, that would be sufficient.

**Comment by Ruizhuo-Xu** [comment]
> Okay, but we are currently preparing for a deadline on Friday. We will release the model before the weekend at the latest.

Thank you, that would be sufficient.
